### PR TITLE
Added banana muffins recipe

### PR DESCRIPTION
**Pull request by tleivy**
teashru zvalim

### CODE REVIEW

**Comment by JuloWaks** [comment]
Please change the file format to MarkDown and update the respective indices

**Comment by tleivy** [comment]
> Please change the file format to MarkDown and update the respective indices

roger

**Comment by JuloWaks** [comment]
indices are the files that show what recipe in each folder

File: dessert/banana_muffins.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<li> 3 cups milk
+<li> 2 cups oat flour
+<li> 2 cups fine cut oats
+<li> 1 cup thick cut oats
+<li> 1 teaspoon baking powder
+<li> 1 cup walnuts
+<li> 100gr chocolate, chopped <br>
+<br>
+1. Mash the bananas <br>
+2. Mix all ingredients <br>
```
Comment (dp25) [inline dessert/banana_muffins.md:16]: what is the order ??

**Review by Rc93s — APPROVED** [review]

**Review by dp25 — COMMENTED** [review]
Nice, however this is not accurate

**Review by dp25 — COMMENTED** [review]
Nice, however this is not accurate

**Review by dp25 — COMMENTED** [review]
Nice, however this is not accurate

**Review by dp25 — COMMENTED** [review]
Nice, however this is not accurate

**Review by dp25 — COMMENTED** [review]
Nice, however this is not accurate

File: dessert/banana_muffins.md (hunk 1 above)
Comment (tleivy) [reply dessert/banana_muffins.md:16]: there is no order :) i would start with the bananas

Comment (tleivy) [reply dessert/banana_muffins.md:16]: there is no order :) i would start with the bananas

**Review by dp25 — APPROVED** [review]
Nice Job TOmerico

**Review by dp25 — APPROVED** [review]
Nice Job TOmerico

**Pull request merged**